### PR TITLE
Skycoin AMA Log (held at ark.io).de.md

### PR DESCRIPTION
**Pull request by Asgaror**

### CODE REVIEW

File: content/ask-the-developers/Skycoin AMA Log ark.de.md [hunk 1]
```diff
@@ -0,0 +1,976 @@
...
+Wann wird es eine Lispcoin-Plattform geben :slightly_smiling_face: ?
+
+**mike**
+Golang ist fertig? Klasse!
+
+**boldninja**
+https://github.com/ArkEcosystem/ark-go GitHub ArkEcosystem/ark-go ark-go - Ark GO client für ARK.io blockchain Ökosystem #golang #ark #blockchain
+
+**mike**
+Lisp - das wäre ein Spaß! Bin immernoch fasziniert, dass Lisp immernoch gibt, aber klar - es ist beliebt für Anwendungen des maschinelles Lernens. Ja, wir brauchen Lisp hinzugefügt, kannst du das machen arc-over-water?
```
Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: Ja, wir sollten Lisp hinzufügen,...statt ..wir brauchen Lisp hinzugefügt

File: content/ask-the-developers/Skycoin AMA Log ark.de.md [hunk 2]
```diff
@@ -0,0 +1,976 @@
...
+**arc-over-water**
+Nun ja, es gibt Shen.. http://shenlanguage.org/.
+
+**synth**
+
+> Was glaubst für das unfassbare Potential für illegale Aktivitäten die Zukunft von Skycoin und der Massenadoption beeinflussen?
+
+Die Drogenhändler werden Skycoin lieben. Bitcoin war wertlos, bis die Leute herausgefunden haben, dass sie damit Drogen verkaufen können, dann erst wurde es zu Geld und ist von $0.01 zu $1000 innerhalb eines Jahres gestiegen.
+Als das Internet erschaffen wurde war das erste was die Leute taten, Pornos herunterzuladen. Als Bitcoin erschaffen wurde war die das erste was die Leute taten, Drogen und Waffen zu kaufen.
+ISIS benötigt Skycoin nicht, sie verwenden XBOX-Messenger und Facebook-Messenger. Sie brauchen Crypto nicht. Jeder terroristische Akt bedeutet Erhöhen der Budgets, deshalb liebt die Regierung ISIS. Terrorismus ist eine Wirtschaftskraft und je mehr Terror, umsro mehr Geld ist dort zu holen. Die Leute machen zuviel Geld durch ISIS, als dass sie das Problems in nächster Zeit beseitigen werden.
```
Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: ...Terror, umso mehr... statt umsro

File: content/ask-the-developers/Skycoin AMA Log ark.de.md [hunk 3]
```diff
@@ -0,0 +1,976 @@
...
+**synth**
+Man hat Mama und Papa, die lokalen Polizeiämter, welche für $150,000 Stingray Handyüberwachungsequipment kaufen auf NSA-Level, welche das eigentlich nicht bezahlen könnten, aber es wird aus den 10 Milliarden DHS "Terror"-Fund finanziert, um nutzloses Equipment für die lokale Polizei zu kaufen. Um "Terrorismus zu bekämpfen". Es ist ein regelrechter Witz.
+
+**arc-over-water**
+SKYs Ziele über die nächsten zwei Jahre?
+Die Preise der nächsten ICO und wie wird sie veröffentlicht?
+
+**synth**
+Die Leute wisen, dass Unternehmen wie Apple und Google obligatorisch dazu veranlagt sind, all ihre Daten der Regierung auszuhändigen. Eingeschlossesn der Nachrichten, also warum würde irgendjemand iMessenger oder iCloud verwenden?
+Sie kreieren also vorgespielte Events wie "Polizei kann nicht den 4-stelligen Pin auf einem iPhone knacken, es ist unknackbar", wobei sie das Handy vom Betreiber aus per Trojaner infizieren können und verschiedenste Arten von Sachen machen können, wir verrückte, obligatorische Hintertüren.
```
Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: ...,wie verrückte, ... statt ...wir verrückte

File: content/ask-the-developers/Skycoin AMA Log ark.de.md [hunk 4]
```diff
@@ -0,0 +1,976 @@
...
+**synth**
+> @synth Was ist dein Background? Kannst du mir etwas über deine bisherige Erfahrung, unabhängig des Gebietes geben?
+
+Ich habe eine Menge Dinge getan. Videospiele, Hedge-Fond/Finanzwesen, Crypto, Investmentzeug etc... Jetzt bin ich eine Art Projektmanager.
+
+**charles**
+Wie wird der Preis der Skycoin für die nächste ICO bestimmt werden, ich vermute es kann nicht überhalb des Marktpreises sein, denn sonst werden die Leute es nicht kaufen, aber zu tief würde den Coin abwerten?
+
+**arc-over-water**
+Wollen sie überhaupt e
```
Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: Bitcoin-Leute, statt Bitcoin-Leuten,
...
Twitter identifiziert haben, statt identifieziert

Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: ...kontinuierliche,... statt kontinuirliche

Comment (Doigmann) [inline content/ask-the-developers/Skycoin AMA Log ark.de.md]: (wir sind seit Jahren mit diesen Leuten.... statt dieses Leute

File: content/ask-the-developers/Skycoin AMA Log ark.de.md [hunk 5]
```diff
@@ -0,0 +1,976 @@
...
+Sieht so aus als hättest du die Frage oben schon beantwortet während ich am Schreiben war...
+
+**tranzer**
+Wie viele Transaktionen pro Sekunde (tx/s) kann Skycoin verarbeiten? Wie lange ist die Blockdauer?
+
+**thrice.pi**
+300 richtig? ^
+
+**arc-over-water**
+Auf eurer Website sagt ihr, ihr werden eine nicht-turing-vollständigen Lisp-Dialekt haben?
```
[the remark this answers is not in the document]
Comment (Asgaror) [reply content/ask-the-developers/Skycoin AMA Log ark.de.md]: touring-vollständig/touring-complete is a specific term, that describes a programming language capable of passing the turing test, i.e. it is possible to copy every program with the specific programming language.

**Comment by Asgaror** [comment]
Thanks for reviewing! 
Worked in all improvements and checked the whole translation one more time afterwards and made some changes. These improvements can be easily viewed by changing the view in 'files changed' between the last two commits.

**Comment by LunaticFringe1** [comment]
Need payment address

**Comment by Asgaror** [comment]
2CA7its9SfYv6P9d34VpdGC62d1rLduVvkF

**Comment by LunaticFringe1** [comment]
12 sky

**Pull request merged**